src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -157,21 +157,29 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
         return Error{*status};
 
     auto const lgrInfo = std::get<ripple::LedgerHeader>(lgrInfoOrStatus);
-    auto const ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
-
-    if (!ledgerObject || ledgerObject->empty())
-        return Error{Status{"entryNotFound"}};
-
+    std::optional<data::Blob> ledgerObject;
+    auto output = LedgerEntryHandler::Output{};
+    output.deletedLedgerIndex = std::nullopt;
+    ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
+    if (!ledgerObject || ledgerObject->empty()) {
+        if (input.includeDeleted == false) {
```

Comment:
Another nit: `if (not input.includeDeleted)` reads better and is shorter. 
When it's possible to make code read like plain English it is usually better to go for it